packages/component/src/Attachment/Text/private/MarkdownTextContent.tsx

```diff
@@ -41,6 +41,10 @@ function isCitationURL(url: string): boolean {
   return onErrorResumeNext(() => new URL(url))?.protocol === 'cite:';
 }
 
+function getEntryName(entry: Entry) {
+  return entry.claim?.appearance?.usageInfo?.name;
```

Comment:
Why only add a function for `getEntryName` and not things like `getEntryTitle` or `getEntryDescription`?